workspaces/xtermjs/Dockerfile

```diff
@@ -1,11 +1,12 @@
 FROM node:22-bookworm
 
-RUN apt-get update && apt-get upgrade -y
-RUN apt-get install -y emacs-nox vim tmux
+RUN apt-get update && apt-get upgrade -y && \
+  apt-get install -y --no-install-recommends emacs-nox vim tmux && \
+  apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
Nit: consistent multiline splits and indentation?

Reply:
I resolved the multiline splits in 0d1081b. I would prefer to wait for indentation till we can use an autoformatter.